source/standards/programming-languages.html.md.erb

```diff
@@ -59,18 +58,16 @@ Our core languages for backend development are:
 - [Java](/manuals/programming-languages/java.html)
 - [Python](/manuals/programming-languages/python/python.html)
 - [Ruby](/manuals/programming-languages/ruby.html)
+- [JavaScript / Node.js](/manuals/programming-languages/nodejs/index.html)
```

Comment:
Open for challenge on this... I never know what to call the thing. Do we refer to Language here -> `JavaScript` or Superset `TypeScript` or runtime `Node.js`

I use all three relatively interchangably in speech, but should probably be more rigerous in docs... the slash feels like a cop-out.

Perhaps JavaScript is  most right?

Reply:
TypeScript / Node.js is better, particularly as a backend language.  It's unlikely we'll want much regular JavaScript.